core/src/main/java/io/questdb/cairo/sql/RowCursorFactory.java

```diff
@@ -25,11 +25,12 @@
 package io.questdb.cairo.sql;
 
 import io.questdb.cairo.TableReader;
+import io.questdb.griffin.Plannable;
 import io.questdb.griffin.SqlException;
 import io.questdb.griffin.SqlExecutionContext;
 import io.questdb.std.ObjList;
 
-public interface RowCursorFactory {
+public interface RowCursorFactory extends Plannable {
```

Comment:
are there RecordFactories that are _not_ plannable ?

Reply:
Factories should all be plannable . 
A number of record cursors is also plannable because they hold important plan info hidden from factories .

Reply:
understood, thanks